frontend/src/assets/styles/styles.scss

```diff
@@ -1,9 +1,22 @@
+@use 'sass:map';
+@use '@bcgov-nr/nr-fsa-theme/design-tokens/type-family.scss' as typeFamily;
+
 // Import all of Bootstrap's CSS
 @import '~bootstrap/scss/bootstrap';
 
+@import '../../../node_modules/primeicons/primeicons.css';

```

Comment:
Just for reference:
I think it is better not to use "../../.", however, if after build and packaged and it works, we can leave it for now.
It should be possible define a alias path for it.
Actually, if looking at line 5 for the '~bootstrap/scss/bootstrap', it is from "node_module/bootstrap" I believe.
And the configuration is probably set in vite.config.ts previously when Vue project was setup.
So if we are not using "bootstrap" we can remove it from configuration and the import and make an alias path to node_module. (but in my limited knowledge, vite.config.ts probably already has a "default" one to node_module. You will need to check if you like to change it.